### ISSUE TITLE
systemctl --user start bluechi-agent as root failed

### ISSUE THREAD

**Issue by dacon42935** [containers/bluechi#561]
## Describe the bug
[root@root ~]# systemctl --user start bluechi-agent
returns
Failed to connect to user bus: No such file or directory
centos 9
root is the only user

## To Reproduce

[root@root ~]# systemctl --user start bluechi-agent
returns
Failed to connect to user bus: No such file or directory

## Expected behavior
to be able to access user bus and be able to list user units from the remote laptop

**Comment by engelmi** [comment]
Hi @dacon42935,
as far as I know root doesn't has a user bus (`$DBUS_SESSION_BUS_ADDRESS` is not set), so you'd have to use the system bus - which is also the recommended way to set up BlueChi. If you really want to use a user bus, then you need to create a new user. 
Is there a reason to connect the BlueChi agent to a user bus instead of the system one?

**Comment by alexlarsson** [comment]
Yeah, you need logged in to a context where you have a full systemd --user instance running for your user, as welll as a dbus --user bus running. This is the case for the *real* user in a regular desktop session. However, it is not the case if you just become root in a terminal.
